README.md

```diff
@@ -15,6 +15,9 @@ Optionally, you may set the `log_group_prefix` parameter to true
 which will scan for all log groups matching the specified prefix(s)
 and ingest all logs available in all of the matching groups.
 
+### Installation
+Run `bin/logstash-plugin install logstash-input-cloudwatch_logs` from your logstash directory
```

Comment:
I needed to run `sudo /usr/share/logstash/bin/logstash-plugin install logstash-input-cloudwatch_logs`.

Maybe you could specify that:
- you might need to use `sudo`
- the logstash directory is likely to be `/usr/share/logstash/`

Btw. even with the current shape, this info is very useful because first I tried each of the following commands and each one failed:
- `sudo /usr/share/logstash/bin/logstash-plugin install logstash-input-cloudwatch-logs` (the first guess - the same as the repo name)
- `sudo /usr/share/logstash/bin/logstash-plugin install input-cloudwatch-logs`
- `sudo /usr/share/logstash/bin/logstash-plugin install cloudwatch-logs`
- `sudo /usr/share/logstash/bin/logstash-plugin install cloudwatch_logs` (the same as the plugin name in the config)

Finally I got the right plugin name from https://rubygems.org/gems/logstash-input-cloudwatch_logs

Then I wanted to open a PR with installation instructions but I found this PR.